app-next/app/movies/page.jsx

```diff
@@ -26,6 +26,8 @@ export default function ExplorePage() {
   const [favorites, setFavorites] = useState([]);
   const [currentPage, setCurrentPage] = useState(1);
   const [totalPages, setTotalPages] = useState(1);
+  const { data: session } = useSession();
+  const user = session?.user;
```

Comment:
The app is using storing user in cookies, and not in session. You can look at a few other places like in actions (like use of getUserProfile method) to see how the user is being retrieved.